src/main/java/com/skillstorm/spartanwireless/dtos/PhoneLineResponseDto.java

```diff
@@ -13,6 +13,4 @@
 public class PhoneLineResponseDto {
     
     private String phoneNumber;
-    private Customer customer;
-    private Device device;
```

Comment:
Are we worried that removing Device from the Response would limit the ability to display the associated Device with the Customer Phone Line on the Front-End?

Reply:
This is a circular dependency issue that had to be resolved/

Reply:
I will have to work on the device customer phone line dependecies. I might do on next pr though.